common/messaging/kafka/kafkaConsumer.go

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
2020

Reply:
it was copied from the previous file that's why is 2017 :D